sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -193,6 +194,36 @@ abstract class JdbcDialect extends Serializable with Logging{
     case _ => value
   }
 
+  /**
+   * Converts aggregate function to String representing a SQL expression.
+   * @param aggregates The aggregate functions to be converted.
+   * @return Converted value.
+   */
+  @Since("3.3.0")
+  def compileAggregates(aggregates: Seq[AggregateFunc]): Option[Seq[String]] = {
```

Comment:
since this is an API now, shall we make it `def compileAggregates(func:AggregateFunc): Option[String]` so that it's cleaner and easier to implement?